ui/server/src/main/scala/pl/touk/nussknacker/ui/process/ProcessTypesForCategories.scala

```diff
@@ -20,4 +20,7 @@ class ProcessTypesForCategories(config: Config) {
     categoriesToTypesMap.get(category)
   }
 
+  def getSystemCategories(): List[String] = {
```

Comment:
why System? maybe just allCategories? And let's make it val

Reply:
Changed - but, why not System? :)

Reply:
we don't use this word so far - and we already have *a lot* of vocabulary :)

Reply:
Okej okej :) Now you can give me approve!